src/test/java/com/wikia/webdriver/common/dataprovider/mobile/MobileAdsDataProvider.java

```diff
@@ -434,18 +394,6 @@ private MobileAdsDataProvider() {
     };
   }
 
-  @DataProvider
-  public static Object[][] mercuryConsecutivePageViews() {
```

Comment:
This test case is quite crucial - we used to have P2s connected to broken things on consecutive page views.

Reply:
And it's here: https://github.com/Wikia/selenium-tests/pull/2195/files#diff-db07ba1d377bf09c465a3fee82235964R81